src/expandableCalendar/index.tsx

```diff
@@ -94,7 +98,7 @@ const headerStyleOverride = {
  * @example: https://github.com/wix/react-native-calendars/blob/master/example/src/screens/expandableCalendar.js
  */
 
-const ExpandableCalendar = (props: ExpandableCalendarProps) => {
+const ExpandableCalendar: ForwardRefRenderFunction<ExpandableCalendarRef, ExpandableCalendarProps> = (props, ref) => {
```

Comment:
Why not simply wrap with 'forwardRef'? What do we need `ForwardRefRenderFunction<ExpandableCalendarRef, ExpandableCalendarProps> ` for?